component/common/src/main/java/org/wso2/carbon/identity/smsotp/common/util/Utils.java

```diff
@@ -128,6 +128,13 @@ private static void sanitizeAndPopulateConfigs(Properties properties) throws SMS
                 Integer.parseInt(otpResendThrottleIntervalValue) * 1000 : Constants.DEFAULT_RESEND_THROTTLE_INTERVAL;
         configs.setResendThrottleInterval(resendThrottleInterval);
 
+        String otpmaxValidationAttemptsAllowedValue = StringUtils.trim(
+                properties.getProperty(Constants.SMS_OTP_MAX_VALIDATION_ATTEMPTS_ALLOWED));
+        int maxValidationAttemptsAllowed = StringUtils.isNumeric(otpmaxValidationAttemptsAllowedValue) ?
+                Integer.parseInt(otpmaxValidationAttemptsAllowedValue) :
+                Constants.DEFAULT_MAX_VALIDATION_ATTEMPTS_ALLOWED;
+        configs.setMaxValidationAttemptsAllowed(maxValidationAttemptsAllowed);
+
```

Comment:
Lets add a comment what this is.

Reply:
Fixed with 13c0ee012c419920360c82927c06d3f0d20ad8c8